buildSrc/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -14,115 +14,133 @@ import org.jooq.meta.jaxb.EmbeddableDefinitionType
-        EnumTable("source_plant_origins", ".*\\.source_plant_origin_id"),
-        EnumTable("species_problem_fields", "species_problems\\.field_id"),
-        EnumTable("species_problem_types", "species_problems\\.type_id"),
-        EnumTable(
-            "storage_conditions",
-            listOf("accessions\\.target_storage_condition", "storage_locations\\.condition_id")),
-        EnumTable("timeseries_types", "timeseries\\.type_id"),
-        EnumTable("upload_problem_types", "upload_problems\\.type_id"),
-        EnumTable(
-            "upload_statuses",
-            listOf("uploads\\.status_id"),
-            "UploadStatus",
-            listOf(EnumTableColumnInfo("finished", "Boolean", false))),
-        EnumTable(
-            "upload_types",
-            "uploads\\.type_id",
-            additionalColumns = listOf(EnumTableColumnInfo("expire_files", "Boolean"))),
-        EnumTable("user_types", ".*\\.user_type_id"),
-        EnumTable("viability_test_seed_types", "viability_tests\\.seed_type_id"),
-        EnumTable("viability_test_substrates", "viability_tests\\.substrate_id"),
-        EnumTable("viability_test_treatments", "viability_tests\\.treatment_id"),
-        EnumTable("viability_test_types", listOf("viability_tests\\.test_type")),
-        EnumTable("withdrawal_purposes", "withdrawals\\.purpose_id"),
+                EnumTable(
+                    "accession_quantity_history_types",
+                    "accession_quantity_history\\.history_type_id"),
+                EnumTable(
+                    "accession_states",
+                    listOf(
+                        "accessions\\.state_id",
+                        ".*\\.accession_state_id",
+                        "accession_state_history\\.(old|new)_state_id"),
+                    additionalColumns = listOf(EnumTableColumnInfo("active", "Boolean"))),
+                EnumTable("collection_sources", ".*\\.collection_source_id"),
+                EnumTable("data_sources", ".*\\.data_source_id"),
+                EnumTable("processing_methods", "accessions\\.processing_method_id"),
+                EnumTable("seed_quantity_units", listOf(".*\\_units_id"), "SeedQuantityUnits"),
+                EnumTable("source_plant_origins", ".*\\.source_plant_origin_id"),
+                EnumTable(
+                    "storage_conditions",
+                    listOf(
+                        "accessions\\.target_storage_condition",
+                        "storage_locations\\.condition_id")),
+                EnumTable("viability_test_seed_types", "viability_tests\\.seed_type_id"),
+                EnumTable("viability_test_substrates", "viability_tests\\.substrate_id"),
+                EnumTable("viability_test_treatments", "viability_tests\\.treatment_id"),
+                EnumTable("viability_test_types", listOf("viability_tests\\.test_type")),
+                EnumTable("withdrawal_purposes", "withdrawals\\.purpose_id"),
```

Comment:
guessing nursery as a withdrawal purpose would be part of this enum - even though it has nursery semantics.

Reply:
It already is, in fact! Which will be a compatibility thing we'll have to figure out.